terra/Dockerfile

```diff
@@ -1,7 +1,6 @@
 # This is a multi-stage docker file:
 #  1. The first stage builds the contracts
 #  2. The second is an empty image with only the wasm files (useful for exporting)
-#  3. The third creates a node.js environment to deploy the contracts to devnet
```

Comment:
why was this dropped?

Reply:
This stage gets broken out to it's own Dockerfile in the next PR in this stack, so the cosmwasm compile container can be shared by terra2/xpla/wormchain. Idk why this line gets removed here, must have been by accident, but it gets sorted in the next one.